util/dvsim/doc/testplanner.md

```diff
@@ -0,0 +1,285 @@
+---
+title: "testplanner tool"
+---
+
+testplanner is a Python based tool for parsing testplans written in Hjson
+format into a data structure that can be used for:
+* Expanding the testplan inline within the DV document as a table
```

Comment:
```suggestion
* Expanding the testplan inline within the DV document as a table;
```

Reply:
Well, sentences need punctuation: choose any that makes sense, but typically only titles don't have one at the end.

Reply:
Okay, done.